src/weatherreport/src/weatherreport_check_nodes_connected.erl

```diff
@@ -0,0 +1,58 @@
+%% -------------------------------------------------------------------
```

Comment:
It would be nice to remove the riaknostic header from all of these checks, especially because it's pretty clear these are checks fully written by Cloudant/IBMers.

Reply:
Actually, I think all the checks containing `riaknostic` (including this file) are originally from Basho, while just the following were contributions from Cloudant:
- weatherreport_check_custodian.erl
- weatherreport_check_internal_replication.erl
- weatherreport_check_ioq.erl
- weatherreport_check_mem3_sync.erl
- weatherreport_check_message_queues.erl
- weatherreport_check_node_stats.erl
- weatherreport_check_process_calls.erl
- weatherreport_check_process_memory.erl
- weatherreport_check_safe_to_rebuild.erl
- weatherreport_check_search.erl
- weatherreport_check_tcp_queues.erl

I'm guessing we want to keep the `riaknostic` headers for the original Basho code, right?